lib/src/linting/README.MD

```diff
@@ -0,0 +1,245 @@
+# Linting
+
+Spot linters are a set of functions whose goal is to ensure a contract definition conforms with our API guidelines. In other words, linters check whether or not a contract is semantically correct.
```

Comment:
```suggestion
Spot linters are a set of functions whose goal is to ensure a contract definition conforms with our API guidelines.
```
I'm not sure we should talk about `semantically correct`. Linters could just describe an API guideline that is just representing a standard at Airtasker. Which at any point in time could be a workaround for a certain situation